index.html

```diff
@@ -0,0 +1,459 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars" id="menu-button">
+        <i class="fas fa-bars"></i>
+      </button>
+      <nav id="desktop-nav" class="desk-nav">
+        <ul>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <div class="menu hide" id="menu-id">
+      <button id="close">
+        <i>X</i>
+      </button>
+      <nav>
+        <ul>
+          <li><a href="#portfolio" class="menu-item">Portfolio</a></li>
+          <li><a href="#about" class="menu-item">About</a></li>
+          <li class="last"><a href="#contact" class="menu-item">Contact</a></li>
```

Comment:
- [ ] Kindly check your links, it's not working properly only the contact link is working other two links are not working please fix this.  

- [ ] Please check the Figma design again your design does not look like the Figma design. Please use the same font family used in the Figma design. Check the screen shoot below and compare it: 

**Your Design:**
![Screenshot from 2023-02-09 16-42-33](https://user-images.githubusercontent.com/45226848/217811879-853a3932-a8fd-467a-80cf-6bfe561f4fea.png)

**Figma Design:**
![Screenshot from 2023-02-09 16-42-13](https://user-images.githubusercontent.com/45226848/217812060-f1f82382-6516-4760-b5e8-b9fce5261a88.png)